policy/modules/system/init.te

```diff
@@ -294,6 +300,16 @@ ifdef(`init_systemd',`
 	# /memfd:systemd-state
 	fs_tmpfs_filetrans(init_t, init_runtime_t, file)
 
+	allow init_t init_var_lib_t:dir manage_dir_perms;
+	allow init_t init_var_lib_t:file manage_file_perms;
+	allow init_t init_var_lib_t:lnk_file manage_lnk_file_perms;
+
+	allow init_t init_tmp_t:dir { manage_dir_perms relabel_dir_perms };
+	allow init_t init_tmp_t:file { manage_file_perms relabel_file_perms };
+	allow init_t init_tmp_t:lnk_file { manage_lnk_file_perms relabel_lnk_file_perms };
+	allow init_t init_tmp_t:sock_file { manage_sock_file_perms relabel_sock_file_perms };
+	files_tmp_filetrans(init_t, init_tmp_t, { dir file })
```

Comment:
this should only apply to dirs (not file, lnk_file, sock_file) AFAIK. also `relabel perms` shouldnt be needed

Reply:
Certainly this access can be reworked. I added this primarily because I noticed some denials related to services with `PrivateTmp=yes` that look like the following:
```
time->Sat Mar 27 12:04:08 2021
type=AVC msg=audit(1616861048.420:31678): avc:  denied  { relabelfrom } for  pid=1 comm="systemd" name="tmp" dev="dm-9" ino=917506 scontext=system_u:system_r:init_t:s0 tcontext=system_u:object_r:tmp_t:s0 tclass=dir permissive=1
----
time->Sat Mar 27 12:04:08 2021
type=AVC msg=audit(1616861048.420:31679): avc:  denied  { relabelto } for  pid=1 comm="systemd" name="tmp" dev="dm-9" ino=917506 scontext=system_u:system_r:init_t:s0 tcontext=system_u:object_r:tmp_t:s0 tclass=dir permissive=1
```
Yet I'm not really sure what is happening here other than these are visible when services are started or stopped.